composer.json

```diff
@@ -52,10 +51,10 @@
     "autoload": {
         "classmap": [
             "scripts/composer/ScriptHandler.php"
-        ]
+        ],
+        "files": ["load.environment.php"]
```

Comment:
We're not using the `load.environment.php` and it's not in the repo, so I assume we dont' need this, unless I\m missing something?

Reply:
I'm just following the drupal-project. So either removing it here or adding the file :) Otherwise it might break

Reply:
it broke, we'll add it :)